node/src/actors/chain_manager/mod.rs

```diff
@@ -1485,6 +1486,10 @@ impl ChainManager {
                         .superblock_state
                         .set_current_superblock(superblock.clone());
 
+                    // Remove superblock beacon target in SessionsManager
+                    let sessions_manager_addr = SessionsManager::from_registry();
+                    sessions_manager_addr.do_send(SetSuperBlockTargetBeacon {beacon: None});
```

Comment:
Why `None` here, if there is consensus?

Reply:
When you put None, you will use your own SuperBlock Beacon, so if your are in consensus, your beacon is the same than the consensus beacon

Reply:
True, let's add a comment explaining that